bin/install

```diff
@@ -51,37 +51,62 @@ install_elm() {
     esac
 
     local download_url="https://github.com/elm/compiler/releases/download/${version}/binary-for-${platform}-${architecture}-bit.gz"
-  else
+  elif [ "$converted_version" -ge 0180 ]; then
     case "$OSTYPE" in
-      darwin*) platform="darwin" ;;
-      linux*) platform="linux" ;;
+      darwin*) platform="mac" ;;
+      linux*)
+        case "$(uname -m)" in
+          x86_64) platform="linux-64bit" ;;
+          i686) platform="linux-32bit" ;;
+          *) fail "Unsupported architecture" ;;
+        esac
       *) fail "Unsupported platform" ;;
     esac
 
-    case "$(uname -m)" in
-      x86_64) architecture="x64" ;;
-      *) fail "Unsupported architecture" ;;
+    source_path="${install_path}/bin/elm.tar.gz"
+
+    local download_url="https://github.com/elm-lang/elm-platform/releases/download/0.18.0-exp/elm-platform-${platform}.tar.gz"
```

Comment:
The old `elm-lang` organization contains the Linux versions while the newer `elm` organization only has the macOS and Windows versions. For this reason I made this a special case.